cli/medperf/tests/comms/test_auth0.py

```diff
@@ -0,0 +1,73 @@
+import time
+from unittest.mock import ANY
+from medperf.tests.mocks import MockResponse
+from medperf.comms.auth.auth0 import Auth0
+
+import pytest
+
+PATCH_AUTH = "medperf.comms.auth.auth0.{}"
+
+
+@pytest.fixture
+def setup(mocker):
+    mocker.patch(PATCH_AUTH.format("requests.post"), return_value=MockResponse({}, 200))
+
+
+def test_logout_removes_credentials(mocker, setup):
+    # Arrange
+    creds = {"refresh_token": ""}
+    mocker.patch(PATCH_AUTH.format("read_credentials"), return_value=creds)
+    spy = mocker.patch(PATCH_AUTH.format("delete_credentials"))
+
+    # Act
+    Auth0().logout()
+
+    # Assert
+    spy.assert_called_once()
+
+
+def test_token_is_not_refreshed_if_not_expired(mocker):
+    # Arrange
+    creds = {
+        "refresh_token": "",
+        "access_token": "",
+        "token_expires_in": 900,
+        "token_issued_at": time.time(),
+    }
+    mocker.patch(PATCH_AUTH.format("read_credentials"), return_value=creds)
+    spy = mocker.patch(PATCH_AUTH.format("Auth0._Auth0__refresh_access_token"))
+
+    # Act
+    Auth0().access_token
```

Comment:
If I understood correctly, retrieving this property should not be updated if the token hasn't expired. What should happen if it has expired? Maybe we should add a test case for that case as well